Source/Krypton Components/Krypton.Ribbon/Controller/AppButtonController.cs

```diff
@@ -379,9 +383,14 @@ public void KeyPress(Control c, KeyPressEventArgs e)
         /// <param name="ribbon">Reference to owning ribbon instance.</param>
         public void KeyTipSelect(KryptonRibbon ribbon)
```

Comment:
If ribbon can truely be null at this point, then change the parameter to be a nullable type